flink-runtime/src/test/java/org/apache/flink/runtime/operators/testutils/DriverTestBase.java

```diff
@@ -190,18 +190,18 @@ protected void setNumFileHandlesForSort(int numFileHandles) {
         this.numFileHandles = numFileHandles;
     }
 
-    @SuppressWarnings("rawtypes")
+    @SuppressWarnings({"rawtypes", "unchecked"})
     protected void testDriver(Driver driver, Class stubClass) throws Exception {
```

Comment:
```suggestion
    protected void testDriver(Driver driver, Class<? extends S> stubClass) throws Exception {
```
I guess, the trick to avoid adding another `\@SuppressWarnings` is to have the generic class being specified in the method signature. :thinking:  WDYT?